src/components/Recommend/Recommend.style.tsx

```diff
@@ -72,3 +76,30 @@ export const Icon = styled.img`
   margin-right: 1.25rem;
   min-width: 2rem;
 `;
+
+export const StyledFilledButton = styled(FilledButton)`
+  width: 100%;
+  margin: 0.5rem 0;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    width: initial;
+    margin: 0.75rem 0 1rem 0;
+  }
+`;
+
+export const StyledOutlinedButton = styled(OutlinedButton)`
+  width: 100%;
+  margin: 0.5rem 0;
+
+  &:hover {
+    background-color: ${props =>
+      props.theme.buttons[ButtonType.OUTLINE].backgroundHover};
+    border: ${props =>
+      `1px solid ${props.theme.buttons[ButtonType.OUTLINE].borderHover}`};
+  }
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    width: initial;
+    margin: 0.75rem 0 1rem 0;
+  }
+`;
```

Comment:
The outlined button and the filled button look the same, can you check if that's what we want? (I changed that locally)

Also, consider implementing a container for the button so you don't need to expose styled versions of each one (we shouldn't be styling the outlined button here, we should use the hover style already implemented)